CMakeLists.txt

```diff
@@ -5,8 +5,7 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
the README mentions "CMake v3.16+", maybe this could be changed to `CMake` instead ? And i assume the book doesn't mention the version anywhere?

Reply:
The book includes code from the qml_minimal CMakeLists.txt which shows the line setting the minimum CMake version.